languages/elixir/exercises/concept/anonymous-functions/.docs/instructions.md

```diff
@@ -0,0 +1,82 @@
+In this exercise, you've been tasked with writing the software for an excryption device that works by performing transformations on data. You need a way to flexibly create complex functions by combining simpler functions together.
```

Comment:
I came across this PR while looking for inspiration for new concept exercises. "complex functions" might be a bit misleading on the first read because it could also refer to [complex functions](https://en.wikipedia.org/wiki/Complex_analysis#Complex_functions) in the mathematical sense, especially in a relatively mathsy context like encryption. Not sure if this is a potential issue on the Elixir track but I thought I'd bring it up anyway.

The exercise and narrative sound fun :)